dsgrid/registry/project_registry_manager.py

```diff
@@ -382,7 +386,7 @@ def _register_all_in_one_dimensions(
             else:
                 suffix = "s"
             dt_plural = f"all_{dt_str}{suffix}"
-            dt_plural_dash = f"all-{dt_str}{suffix}"
+            dt_plural_dash = f"all-{model.project_id}-{dt_str}{suffix}"
```

Comment:
This is an unrelated change. I noticed that we had a potential problem. Every project was going to have all-in-one dimensions with very similar names and IDs. This should make it more clear.